content/help/faq/distributed-tracing/how-to-support-tracing.md

```diff
@@ -0,0 +1,19 @@
+---
+title: What is required for Distributed Tracing with Istio?
```

Comment:
"distributed tracing", no need to capitalize. This applies to all FAQ title: entries.